src/components/DataTable/data-table.scss

```diff
@@ -37,6 +37,10 @@
         top: 50%;
         transform: translateY(-50%);
       }
+
+      br {
```

Comment:
Was it added because the Slot Story? You could remove the `<br />` tag from the Story, not in CSS. What do you think?

Reply:
This behavior is for the loading. When prop isLoading is true, I need to hide the content inside the cells and i'm doing this with css. Thinking better, I'll change to * instead of <br>.

Reply:
Ok, understood!